Framework/Algorithms/inc/MantidAlgorithms/SampleCorrections/IBeamProfile.h

```diff
@@ -8,9 +8,11 @@
 
 #include "MantidAlgorithms/DllConfig.h"
 #include "MantidGeometry/Objects/BoundingBox.h"
+#include "MantidGeometry/Objects/IObject.h"
 #include "MantidKernel/V3D.h"
 
 namespace Mantid {
+using Kernel::V3D;
```

Comment:
Please move `using` out of the header. It pollutes the namespace of everything that imports this.